////checked exception2 handling using try and catch

public class ArgEx {
    public static void main(String[] args) throws InterruptedException{
        Thread.sleep(2000);
    }
    
}
